src/client.ts

```diff
@@ -635,25 +714,46 @@ class OptimizelyReactSDKClient implements ReactSDKClient {
     overrideUserId?: string,
     overrideAttributes?: optimizely.UserAttributes
   ): VariableValuesObject {
+    if (!this._client) {
+      logger.warn(
+        'Unable to get feature variables for feature "%s" because Optimizely client failed to initialize.',
+        featureKey
+      );
+      return {};
+    }
+
     const user = this.getUserContextWithOverrides(overrideUserId, overrideAttributes);
+
     const userId = user.id;
     if (userId === null) {
-      logger.info('getFeatureVariables returning `{}` for feature "%s" because userId is not set', featureKey);
+      logger.warn('Unable to get feature variables for feature "%s" because User ID is not set', featureKey);
       return {};
     }
+
     const userAttributes = user.attributes;
     const variableObj: VariableValuesObject = {};
+
     const optlyConfig = this._client.getOptimizelyConfig();
     if (!optlyConfig) {
+      logger.warn(
+        'Unable to retrieve feature variables for feature "%s" because Optimizely client failed to initialize.',
+        featureKey
+      );
       return {};
     }
+
     const feature = optlyConfig.featuresMap[featureKey];
     if (!feature) {
+      logger.warn(
```

Comment:
I would suggest to convert that to `info`. Its probably not an unusual behavior.